### PR TITLE
Song Selector UI

### PR DESCRIPTION
**Pull request by epeach**
Behind experiment flag "enableExperiments=songSelector"

*Adds a dropdown UI element above the visualization in Dance Lab levels
*Adds songLibrary.json - placeholder while song storage is still in development. Chosen to emulate soundLibrary storage and retrieval.
*Adds simple Game Lab UI test

Does not:
*Adjust spacing of Sprite Lab beta dialog - confirmed with Ryan that this will be removed by the time song selector is removed
*Apply song choice

Screenshot:
screenshot from 2018-09-21 11-41-21

cc: @ryansloan

### CODE REVIEW

File: apps/src/code-studio/songLibrary.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+{
+  "macklemore90": {
```
Comment (davidsbailey) [inline apps/src/code-studio/songLibrary.json:3]: is the `90` a typo?

File: dashboard/test/ui/features/spritelab/dancelab.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Feature: Dance Lab
+  Scenario: Without flag, song selector is not visible
+    Given I am on "http://studio.code.org/s/allthethings/stage/37/puzzle/1?noautoplay=true"
+    And I rotate to landscape
+    And I wait for the page to fully load
+    And I wait for 3 seconds
```
Comment (davidsbailey) [inline dashboard/test/ui/features/spritelab/dancelab.feature:6]: @joshlory what are we waiting to have happen here? would be nice to wait explicitly for this before this pattern gets copied too many times. Erin that shouldn't block your PR though

File: apps/src/code-studio/songLibrary.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  },
+  "hammer": {
+    "title": "MC Hammer - U Can't Touch This",
+    "url": "https://temporary.url"
+  },
+  "peas": {
+    "title": "The Black Eyed Peas - I Got a Feeling",
+    "url": "https://temporary.url"
+  }
+}
```
Comment (davidsbailey) [inline apps/src/code-studio/songLibrary.json:15]: this format seems fine for a placeholder.

File: dashboard/config/scripts/levels/allthethings HOC Dance 1.level [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+          <title name="DIR">-1</title>
+        </block>
+        <block type="Dancelab_atTimestamp">
+          <title name="TIMESTAMP">4</title>
+          <title name="UNIT">"measures"</title>
+        </block>
+      </xml>
+    </toolbox_blocks>
+  </blocks>
+</Dancelab>
```
Comment (davidsbailey) [inline dashboard/config/scripts/levels/allthethings HOC Dance 1.level:249]: 👍 good call to copy this

File: apps/src/code-studio/songLibrary.json (hunk 1 above)
Comment (epeach) [reply apps/src/code-studio/songLibrary.json:3]: It's the 90 second clip of the song.

**Pull request merged**